sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -394,16 +397,18 @@ export interface CreateVectorStoreFileOptions {
 }
 
 // @public
-export interface CreateVectorStoreFileWithPollingOptionalParams extends CreateVectorStoreFileOptions, PollingOptionsParams, OperationOptions {
-}
+export type CreateVectorStoreFileResponse = PromiseLike<VectorStoreFileOutput> & {
+    poller: PollerLike<PollOperationState<VectorStoreFileOutput>, VectorStoreFileOutput>;
+};
```

Comment:
Why is the poller being returned in a field in the response object? I think what we should do is to return a `PollerLike` from the method. `VectorStoreFileOutput` could then be accessed from `poller.operationState.result.body`.

Reply:
I think we want to return a `VectorStoreFileOutput` by default to adhere to the typespec/api contract and then provide the poller in case the user wants to poll. @ganeshyb what do you think?

Reply:
That is correct, by default we return the response, unless the user wants to poll for their scenarios.

Reply:
We discussed this offline and agreed that the long running operations should return `PollerLike` directly.